resources/setup_emulator.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+kill_all_emus() {
+    for emu_device in $(adb devices -l |grep 'device product:' |cut -d' ' -f1); do
+        echo "Killing emulator: $emu_device"
+        adb -s $emu_device emu kill
+    done
+}
+
+echo 'Creating emulator'
+echo 'n' | $ANDROID_HOME/tools/bin/avdmanager create avd -n test-android -f -k "system-images;android-27;google_apis;x86" 
+
+echo 'loading emulator'
+$ANDROID_HOME/emulator/emulator -avd test-android -no-boot-anim -no-window -noaudio & 
+sleep 2m
```

Comment:
^ does it always take around this long? Is there a more definitive way we can check if it has finished loading?

Reply:
Adding loop to wait for emulator availability